api/proto/banyandb/measure/v1/write.proto

```diff
@@ -45,8 +45,23 @@ message WriteRequest {
   DataPointValue data_point = 2 [(validate.rules).message.required = true];
 }
 
+// Status is the response status for write
+enum Status {
+  STATUS_UNSPECIFIED = 0;
+  STATUS_SUCCEED = 1;
+  STATUS_RECEIVE_ERROR = 2;
+  STATUS_INVALID_TIMESTAMP = 3;
+  STATUS_INVALID_METADATA = 4;
+  STATUS_EXPIRED_REVISION = 5;
+  STATUS_INTERNAL_ERROR = 6;
+}
+
 // WriteResponse is the response contract for write
-message WriteResponse {}
+message WriteResponse {
```

Comment:
I would like to define request identification to both request and response.

You can add a unique messageId to both the WriteRequest and WriteResponse messages. With this messageId, we can easily identify which response corresponds to a particular request.